mmv1/third_party/terraform/go.mod.erb

```diff
@@ -1,7 +1,9 @@
 <% autogen_exception -%>
 module github.com/hashicorp/terraform-provider-google
 
-go 1.20
+go 1.21
```

Comment:
We'll need to bump the provider up to Go 1.21!

Reply:
This has been confirmed in the release notes for 1.6.1 : https://github.com/hashicorp/terraform-plugin-framework/releases/tag/v1.6.1

Reply:
See these PRs for progress updating the provider to Go 1.21:
- https://github.com/GoogleCloudPlatform/magic-modules/pull/10169
- https://github.com/GoogleCloudPlatform/magic-modules/pull/10175
- https://github.com/GoogleCloudPlatform/terraform-google-conversion/pull/2043